ActiveRecord/ActiveRecord_vs170.vcxproj

```diff
@@ -230,7 +230,7 @@
   </ImportGroup>
   <PropertyGroup Label="UserMacros"/>
   <PropertyGroup>
-    <_ProjectFileVersion>17.0.34322.80</_ProjectFileVersion>
+    <_ProjectFileVersion>17.0.34511.75</_ProjectFileVersion>
```

Comment:
seems like we're going backwards here? there is no reason to modify other vs projects in this PR